src/server/honeypot.ts

```diff
@@ -10,27 +10,31 @@ export interface HonetpotConfig {
   randomizeNameFieldName?: boolean;
   nameFieldName?: string;
   validFromFieldName?: string;
-  validFromTimestamp?: number;
```

Comment:
This option doesn't work as expected when globally configured, so we do the same thing when calling `getInputProps`.